anitya/default_config.py

```diff
@@ -18,7 +18,8 @@
 SECRET_KEY = 'changeme please'
 
 # URL to the database
-DB_URL = 'sqlite:////var/tmp/anitya-dev.sqlite'
+repo_root = path.abspath(path.dirname(path.dirname(__file__)))
+DB_URL = 'sqlite:///{0}/anitya.db'.format(repo_root)
```

Comment:
@pypingou, I had trouble with Apache reading the database in that location. I was able to get it to work when the database was under /opt or /var/www though.

Anyways, I returned it to the previous value in my latest commit.